spec/messages_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper'
+require './lib/messages'
+
+describe Message do
```

Comment:
how useful are these tests? are they just testing data? It seems like there is one test, one message, one method.  What about a generic method to handle any message? Then the actual message doesn't matter so much.

Reply:
Testing the messages/instructions is where I got really stuck in the weeds last week. For context, I had previously been using a method similar to what you seem to be describing (it was called prompt, and it would accept an argument parameter and print it onto the console). I mentioned this in retro, but I didn't know how to test whether "prompt" was behaving as expected, and I get stuck in the same place when considering how to refactor here as well. I would be down to discuss this during our pairing tomorrow! But if not, would this be a situation that necessitated a test double?

Reply:
we can look into it when we pair tomorrow.

Reply:
Echoing what Ian is getting at - I like housing all the messages in your Message class.  Could be useful to explore a UserInterface and a generic method that you can pass any message into and it will print that message.

Reply:
I like the UserInterface idea! `display_board` (which is currently housed in TicTacToe class) would fit in there better as well.